### PR TITLE
Catchall mappings

### PR DESCRIPTION
**Pull request by NickPapacostas**
Mapping changes for adding a standard analyzed catch all field

### CODE REVIEW

**Comment by NickPapacostas** [comment]
related to: https://github.com/guardian/grid/pull/1745

**Comment by kenoir** [comment]
:+1:

File: common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala [hunk 1]
```diff
@@ -120,7 +121,7 @@ object Mappings {
   val userMetadataMapping =
     nonDynamicObj(
       "archived"    -> boolean,
-      "labels"      -> nonAnalysedList("label"),
+      "labels"      -> (nonAnalysedList("label") ++ copyTo("metadata.englishAnalysedCatchAll")),
```
Comment (theefer) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala]: Looks like I commented on another similar PR, but are we sure want to add labels to this?

Comment (theefer) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala]: See other comments on https://github.com/guardian/grid/pull/1568

**Pull request merged**